easybuild/easyconfigs/r/R-bundle-Bioconductor/R-bundle-Bioconductor-3.16-foss-2022b-R-4.2.2.eb

```diff
@@ -0,0 +1,1240 @@
+    }),
+    ('cummeRbund', '2.40.0', {
+        'checksums': ['a955bc1b76c59d1fab97de09740d524c770f5954a58990a906ed93fda885f3bb'],
+    }),
+    ('GenomicFiles', '1.34.0', {
+        'checksums': ['a11026ba7461816097b889aa1148f4c5d3227b041e1ed4384817215f5caac44c'],
+    }),
+    ('derfinderHelper', '1.32.0', {
+        'checksums': ['0478def20d6861779e4c640283d71759bc3bbf56131cc9db67718fe271951985'],
+    }),
+    ('derfinder', '1.32.0', {
+        'checksums': ['f920adcd310b352a070bbd6d03a2587fb7626add4a74071ac8e2cced3c1c306d'],
+    }),
+    ('polyester', '1.34.0', {
+        'checksums': ['bac7b6a80979d604732e3ccc785ceca08fa37760312abf8332f3ce1ba9278a9b'],
+    }),
+    ('Rsubread', '2.12.3', {
+        'checksums': ['ad498c5c386219989b689e5bd22eb16d7f741d666bbb0009598bc2e791da1552'],
+    }),
+    ('pcaMethods', '1.90.0', {
+        'checksums': ['440fc4441798c50ecd39416188e82375f2484a9732e51197e9f3bbb5ae9b55b2'],
+    }),
+    ('marray', '1.76.0', {
+        'checksums': ['fa4d8751414a3038e0c07dc61b90fc10f6ddcfedd520dbfb365088dd25234dad'],
+    }),
+    ('CGHbase', '1.58.0', {
+        'checksums': ['77b7b07137f4f48208c063d8c1f4390d52f3c4938f2a2747f606125844f9c106'],
+    }),
+    ('Wrench', '1.16.0', {
+        'checksums': ['3d0d9b3127e2622dde59ce78d3ff16fbeebc3e9e3391480c2f1cdd2693dcf558'],
+    }),
+    ('lpsymphony', '1.26.3', {
+        'checksums': ['e57bb9a414cd9bf219072179c58d32715ca0ae4475968145a4a0f04bea340c47'],
+    }),
+    ('IHW', '1.26.0', {
+        'checksums': ['3e7b896a87b2c21a9da91a4a4634bca4b70ea0671557126b71cde3bf04716b13'],
+    }),
+    ('metagenomeSeq', '1.40.0', {
+        'checksums': ['5e26362985890a6bb22b17b755b9d5d3df1293220fb3617551daa3ca26e19207'],
+    }),
+    ('gdsfmt', '1.34.0', {
+        'checksums': ['06cc0c5d087859a897c46bca4555b957df99453bb9d1cbe2dee64ab809c663b2'],
+    }),
+    ('SNPRelate', '1.32.2', {
+        'checksums': ['9daadded411c04ac33b98c27d4d873f735cc757b4ef4874589cfa76091da60c5'],
+    }),
+    ('biomformat', '1.26.0', {
+        'checksums': ['21dac4201474b56610d0aa8c6f4dc0d5514dec6a37897a695176af02e475481c'],
+    }),
+    ('phyloseq', '1.42.0', {
+        'checksums': ['736a9bf5315e548293e41c0c6696c3147357c6abee308ec3a8997fb35301f11f'],
+    }),
+    ('NADA', '1.6-1.1', {
+        'checksums': ['670ff6595ba074ed0a930b7a09624d5ef20616379a20e768c1a7b37332aee44a'],
+    }),
+    ('zCompositions', '1.4.0-1', {
+        'checksums': ['33ee11f635cb87cc9c0617e1cfc91f1ac41c6cfe2b70fc441e226015939230e7'],
+    }),
+    ('RcppZiggurat', '0.1.6', {
+        'checksums': ['9c78255ca476c945c05a564d1e4da363de714d890e0e27f3b252fd73c50eed71'],
+    }),
+    ('Rfast', '2.0.7', {
+        'checksums': ['8f86159a4760a7124e1c91ae0b022c7e496f81590ea4e4af702bea44e8dedf8f'],
+    }),
+    ('ALDEx2', '1.30.0', {
+        'checksums': ['6eba6fdd689fdd5d18fde3ffe4aa64ad4af5bfdc8977b8038d297fb46ed10515'],
+    }),
+    ('dada2', '1.26.0', {
+        'patches': ['dada2-1.16.0-remove-sse-on-aarch64.patch'],
```

Comment:
Patch does not apply cleanly - possibly no longer needed

Reply:
Patch is no longer needed - https://benjjneb.github.io/dada2/ReleaseNotes_1_26.html